src/cache/cache.rs

```diff
@@ -197,7 +200,10 @@ impl CacheWrite {
     where
         T: Read,
     {
-        let opts = FileOptions::default().compression_method(CompressionMethod::Deflated);
+        // We are going to cheat a little bit here; we're going to declare the
```

Comment:
That's not really cheating. Cheating would be using another compression method than stored.

Reply:
I will reword this comment.

Reply:
You didn't.